csharp-intermediate-topics/UnsafeCodeInCSharp/UnsafeCodeInCSharp/Program.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace UnsafeCodeInCSharp
+{
+    public class Program
+    {
+        public static unsafe void Main(string[] args)
+        {
+            unsafe
+            {
+                int a = 100;
+                int* ptr = &a;
+
+                Console.WriteLine("The value of variable: {0}", a);
+                Console.WriteLine("The value of variable using pointer: {0}", ptr->ToString());
+                Console.WriteLine("The address of variable : {0}", (int)ptr);
+            }
+
+            int num = 10;
+            int* arg = &num;
```

Comment:
Could we use a clearer name for this variable instead of `arg`? Maybe `numPointer` or something along those lines?